### PR TITLE
[WIP] : Add FASTQ strategy generation

### PR DESCRIPTION
**Pull request by vaastav**
I have marked it as a Work In Progress as I would like to test if the functions I wrote actually work or not.
This would require writing some unit tests which I probably would get around to later in the week.

### CODE REVIEW

File: hypothesis_bio/hypothesis_bio.py [hunk 1]
```diff
@@ -106,3 +127,24 @@ def fasta(draw):
...
+    fasta_sequence = draw(fasta_source)
+    raw_sequence = fasta_sequence["sequence"]
+    sequence_id = fasta_sequence["comment"]
+    sequence_length = len(raw_sequence)
+    quality_string = draw(quality_source(sequence_length))
+    return (
+        "@"
+        + sequence_id
+        + "\n"
+        + raw_sequence
```
Comment (mbhall88) [inline hypothesis_bio/hypothesis_bio.py]: You need a newline after the `raw_sequence`

File: hypothesis_bio/hypothesis_bio.py [hunk 2]
```diff
@@ -38,6 +38,27 @@ def dna(
...
+    - `size`: Size of the quality string to be generated
+
+    Note:
+    According to https://en.wikipedia.org/wiki/FASTQ_format,
+    the range of characters for the quality string ranges from
+    byte value 0x21 to 0x7E.
+    """
+    return draw(
+        text(
+            alphabet=characters(min_codepoint=33, max_codepoint=121),
```
Comment (Benjamin-Lee) [inline hypothesis_bio/hypothesis_bio.py]: 0x7E is 126 but `max_codepoint=121`. Is there a reason for this?

File: hypothesis_bio/hypothesis_bio.py (hunk 1 above)
Comment (vaastav) [reply hypothesis_bio/hypothesis_bio.py]: Ahh ofc. Fixed it now

File: hypothesis_bio/hypothesis_bio.py (hunk 2 above)
Comment (vaastav) [reply hypothesis_bio/hypothesis_bio.py]: Hmm I think I screwed up with my math I think.

**Comment by Benjamin-Lee** [comment]
I'm merging this since it looks good. We'll have to figure out unit testing eventually, but its nontrivial. We'll probably have to copy paste something like [this](https://github.com/HypothesisWorks/hypothesis/blob/c3cce7b627f10716b448964fb376fe5626b360c6/hypothesis-python/tests/common/debug.py#L34) so that we can reproducibly generate minimal examples to test against.

**Pull request merged**